src/components/common/YourCompany/YourCompany.vue

```diff
@@ -305,6 +316,7 @@ import { CommonMixin, SharedMixin, DateMixin, LegalApiMixin, NameRequestMixin }
 import { CorrectionTypes } from '@/enums/'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module/'
 import { cloneDeep } from 'lodash'
+import { ConversionNOB } from '../../Conversion'
```

Comment:
I think using "local path" is OK (`./`) but anything more should use the normal path specification. In this case, I think `from '@/components/Conversion/'` will work.

Reply:
This will raise an error: `"Cannot find module '@components/Conversion/' or its corresponding type declarations."`. I already exported `ConversionNOB` in the file `@/components/Conversion/index.ts`. The same goes to line 313 and 314, it's using `./` not `@`. It happens in many places, and I had to use local path for `.vue` files. If that's a `.ts` file, then `@` won't raise any error.

Reply:
What if you try from `'@/components/Conversion/'` (slash between @ and components)?

The difference is:
- `@blah` refers to an imported package
- `@/blah` refers to a folder in /src as per https://github.com/bcgov/business-edit-ui/blob/0dd1590dc82f7f29c83dbcf41c80b807d7144e6b/tsconfig.json#L21

Reply:
The whole point of using an index.ts file to export components is so we can import them from "the folder" instead of from the source file itself.

Reply:
Looks good.